src/MahApps.Metro/Styles/Controls.Scrollbars.xaml

```diff
@@ -4,19 +4,19 @@
                     xmlns:System="clr-namespace:System;assembly=mscorlib"
                     xmlns:po="http://schemas.microsoft.com/winfx/2006/xaml/presentation/options">
 
-    <Storyboard x:Key="MetroScrollBarThumbMouseOverStoryboard" po:Freeze="True">
+    <Storyboard x:Key="MahApps.Storyboard.MahApps.Styles.ScrollBar.MetroThumbMouseOver" po:Freeze="True">
```

Comment:
Shouldn't be this like that? -> `MahApps.Storyboard.ScrollBar.MetroThumbMouseOver`